src/frontend/src/store/index.ts

```diff
@@ -367,26 +397,40 @@ export default new Vuex.Store({
       }
 
       // If we have valid fields, send off to user service for account creation
+      const keyPair = genKeyPair();
+      const privateKey = keyPair[0];
+      const walletId = keyPair[1];
+      const privateKeyHash = sha256(`${umnetId}${password}${walletId}`)
+
+      // TODO: Remove when we read in and set privateKey on login
+      commit("MUTATION_SET_PRIVATE_KEY", privateKey)
+      commit("MUTATION_SET_WALLETID", walletId)
 
       axios
         .post("http://localhost/users/create", {
-          "firstName": firstName,
-          "lastName": lastName,
+          "first_name": firstName,
+          "last_name": lastName,
           "umnetID": umnetId,
-          "public_key": getters.walletId,
+          "public_key": walletId,
           "password": password,
         })
         .then(
           () => {
-            commit("MUTATATION_SET_LOADING", false);
+            commit("MUTATION_SET_LOADING", false);
+
+            // TODO: Uncomment when we read in and set privateKey on login
```

Comment:
TODO: Create an issue to tackle reading and writing privateKeys from local text file. Thought it was gonna be obvious but it isn't!